src/Common/src/Interop/Unix/System.Net.Security.Native/SecuritySafeHandles.cs

```diff
@@ -13,16 +13,68 @@ namespace System.Net.Security
     internal sealed class SafeFreeNegoCredentials : SafeFreeCredentials
     {
         private SafeGssCredHandle _credential;
+        private readonly bool _isNtlm;
+        private readonly string _userName;
+        private readonly bool _isdefault;
 
         public SafeGssCredHandle GssCredential
         {
             get { return _credential; }
         }
 
-        public SafeFreeNegoCredentials(string username, string password, string domain) : base(IntPtr.Zero, true)
+        public bool IsNtlm
         {
+            get { return _isNtlm; }
+        }
+
+        public string UserName
+        {
+            get { return _userName; }
+        }
+
+        public bool IsDefault
+        {
+            get { return _isdefault; }
+        }
+
+        public SafeFreeNegoCredentials(bool ntlmOnly, string username, string password, string domain)
```

Comment:
Here it's called "ntlmOnly", but the field / property are called IsNtlm.  Those names could imply different things.  Regardless, they should match, whichever is correct.